desktop/npm-helper.js

```diff
@@ -193,26 +193,53 @@ function fixupSymlinks () {
 function updatedFonts () {
   const project = JSON.parse(fs.readFileSync('./shared/images/iconfont/kb-icomoon-project.json', 'utf8'))
   const sizes = {}
+  const fontNames = {}
+  const fontAwesomeNames = {}
 
   project.iconSets.forEach(set => {
     set.icons.forEach(icon => {
       const name = icon.tags[0]
       if (name.startsWith('kb-')) {
         sizes[`fa-${name}`] = icon.grid
+        fontNames[`'fa-${name}'`] = true
+      } else {
+        fontAwesomeNames[`'fa-${name}'`] = true
       }
     })
   })
 
-  const json = JSON.stringify(sizes, null, 2).replace(/"/g, "'").replace('\n}', ',\n}\n')
-  const data = `// @flow
+  const fontSizes = JSON.stringify(sizes, null, 2).replace(/"/g, "'").replace('\n}', ',\n}\n')
+
+  const flowFontIcons = Object.keys(fontNames).sort().map(f => `${f}: true`).join(',\n  ')
+  const flowFontAwesomeIcons = Object.keys(fontAwesomeNames).sort().map(f => `${f}: true`).join(',\n  ')
+
+  const flowIcons = fs.readdirSync('../shared/images/icons')
+    .filter(f => f.indexOf('@') === -1 && f.startsWith('icon-'))
+    .map(f => `'${f.slice(0, -4)}': true`).join(',\n  ')
+
+  const text = `// @flow
 // This file is GENERATED by npm run updated-fonts. DON'T hand edit
 
-import type {IconType} from './icon'
+export const FontIcons = {
```

Comment:
we now make a list of all the images and fonts, no more hand editing!

proof-new and proof-followed are 'fake' icons. we could discuss ditching this concept